### PR TITLE
Hotfix/elastic/indexer

### PR DESCRIPTION
**Pull request by Urhengulas**

### CODE REVIEW

File: elastic/indexer/insta_comments/Dockerfile [hunk 1]
```diff
@@ -3,12 +3,14 @@ WORKDIR /app
...
+    elastic/indexer/insta_comments/insta_comments_indexer.go
 
 FROM alpine
 RUN apk --no-cache add ca-certificates
-RUN mkdir /app
-COPY useragents.json /app
-COPY --from=builder /app/es-comments-indexer /app
 WORKDIR /app
-CMD ["./es-comments-indexer"]
+COPY useragents.json .
```
Comment (1Jo1) [inline elastic/indexer/insta_comments/Dockerfile]: remove useragents.json

File: elastic/indexer/insta_comments/insta_comments_indexer.go [hunk 2]
```diff
@@ -1,26 +1,29 @@
-package main
+package insta_comments
```
Comment (1Jo1) [inline elastic/indexer/insta_comments/insta_comments_indexer.go]: you have to use the main package to execute the main function

File: elastic/indexer/insta_faces/Dockerfile [hunk 3]
```diff
@@ -3,12 +3,14 @@ WORKDIR /app
...
+    elastic/indexer/insta_faces/insta_faces_indexer.go
 
 FROM alpine
 RUN apk --no-cache add ca-certificates
-RUN mkdir /app
-COPY useragents.json /app
-COPY --from=builder /app/indexer /app
 WORKDIR /app
-CMD ["./indexer"]
+COPY useragents.json .
```
Comment (1Jo1) [inline elastic/indexer/insta_faces/Dockerfile]: remove useragents

File: elastic/indexer/insta_users/insta_users_indexer.go [hunk 4]
```diff
@@ -1,14 +1,16 @@
-package main
+package insta_users
```
Comment (alexmorten) [inline elastic/indexer/insta_users/insta_users_indexer.go]: this should still be the main package

**Review by alexmorten — CHANGES_REQUESTED** [review]

File: elastic/indexer/insta_comments/Dockerfile [hunk 5]
```diff
@@ -3,12 +3,14 @@ WORKDIR /app
 COPY go.mod go.sum ./
 RUN go mod download
 COPY . .
-RUN CGO_ENABLED=0 GOOS=linux go build -installsuffix cgo -o es-comments-indexer elastic/indexer/insta_comments/main/main.go
+RUN CGO_ENABLED=0 GOOS=linux go build \
+    -installsuffix cgo \
+    -o exe \
```
Comment (alexmorten) [inline elastic/indexer/insta_comments/Dockerfile]: let's not use `exe` here

File: elastic/indexer/insta_faces/Dockerfile [hunk 6]
```diff
@@ -3,12 +3,14 @@ WORKDIR /app
...
+COPY --from=builder /app/exe .
+CMD ["./exe"]
```
Comment (alexmorten) [inline elastic/indexer/insta_faces/Dockerfile]: please let's not use `exe` here

File: elastic/indexer/insta_posts/Dockerfile [hunk 7]
```diff
@@ -3,12 +3,14 @@ WORKDIR /app
...
 FROM alpine
 RUN apk --no-cache add ca-certificates
-RUN mkdir /app
-COPY useragents.json /app
-COPY --from=builder /app/es-posts-indexer /app
 WORKDIR /app
-CMD ["./es-posts-indexer"]
+COPY useragents.json .
+COPY --from=builder /app/exe .
+CMD ["./exe"]
```
Comment (alexmorten) [inline elastic/indexer/insta_posts/Dockerfile]: also here

**Review by alexmorten — CHANGES_REQUESTED** [review]

**Review by 1Jo1 — APPROVED** [review]

**Review by alexmorten — APPROVED** [review]

**Pull request merged**